app/src/main/java/com/bashkevich/androidfundamentals/model/MoviesDao.kt

```diff
@@ -0,0 +1,29 @@
+package com.bashkevich.androidfundamentals.model
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+import com.bashkevich.androidfundamentals.model.entity.ActorEntity
+import com.bashkevich.androidfundamentals.model.entity.MovieActorCrossRef
+import com.bashkevich.androidfundamentals.model.entity.MovieEntity
+import com.bashkevich.androidfundamentals.model.entity.MovieWithActors
+
+@Dao
+interface MoviesDao {
+
+    @Transaction
```

Comment:
Why transaction? I don't think you need transaction for the select queries at all

Reply:
Transactions are used in mamy-to-many for selecting from multiple tables

Reply:
Oke, just read that it is useful https://stackoverflow.com/questions/5982517/use-transactions-for-select-statements